Animations/Fly.js

```diff
@@ -1,28 +1,27 @@
 var makeAnimation = require('../src/TuxAnimations');
```

Comment:
Add 'use strict'; on line one and leave a newline between that and variable declarations.